enterprise_access/apps/api/v1/views/subsidy_access_policy.py

```diff
@@ -500,8 +505,14 @@ def evaluate_policies(self, enterprise_customer_uuid, lms_user_id, content_key,
         """
         redeemable_policies = []
         non_redeemable_policies = defaultdict(list)
-        all_policies_for_enterprise = self.get_queryset()
-        for policy in all_policies_for_enterprise:
+        # Sort policies by:
+        # - priority (of type)
+        # - expiration, sooner to expire first
+        # - balance, lower balance first
+        all_sorted_policies_for_enterprise = sort_subsidy_access_policies_for_redemption(
```

Comment:
nit: consider adding a note to the docstring that the redeemable policies returned by this method are sorted in a meaningful way.